pkg/cloudprovider/providers/aws/aws_loadbalancer.go

```diff
@@ -62,6 +64,32 @@ func (s *AWSCloud) ensureLoadBalancer(namespacedName types.NamespacedName, loadB
 		if err != nil {
 			return nil, err
 		}
+
+		// XXX Do we always want to create this policy or should we do this lazily
+		// and create when requested
+		err = s.createProxyProtocolPolicy(loadBalancerName)
+		if err != nil {
+			return nil, err
+		}
+
+		if proxyProtocol {
+			for _, proxyListener := range listeners {
+				// TODO Refactor this out into a seperate method
+				request := &elb.SetLoadBalancerPoliciesForBackendServerInput{
+					InstancePort:     proxyListener.InstancePort,
+					LoadBalancerName: aws.String(loadBalancerName),
+					PolicyNames: []*string{
+						aws.String(ProxyProtocolPolicyName),
+					},
+				}
+				glog.V(2).Info("Enabling proxy protocol policy on node port %d", *proxyListener.InstancePort)
+				_, err := s.elb.SetLoadBalancerPoliciesForBackendServer(request)
```

Comment:
If we did move the logic out of create (as I suggested above), we should probably check that the existing listener does not have this policy before setting it, otherwise we'll likely start hitting AWS rate limits again